bitnami/moodle/templates/deployment.yaml

```diff
@@ -174,23 +241,19 @@ spec:
           readinessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.customReadinessProbe "context" $) | nindent 12 }}
           {{- end }}
           {{- if .Values.resources }}
-          resources: {{ toYaml .Values.resources | nindent 12 }}
+          resources: {{- toYaml .Values.resources | nindent 12 }}
           {{- end }}
           volumeMounts:
             - name: moodle-data
-              mountPath: /bitnami
+              mountPath: /bitnami/moodle
```

Comment:
This will break upgrades. Note that Moodle persists the following directories:

- /bitnami/moodle
- /bitnami/moodledata

You can see how I did it in OpenCart, or keep previous configuration for now.

Reply:
Changed now, thanks for the catch!